fe/fe-core/src/main/java/com/starrocks/analysis/ShowTableStatusStmt.java

```diff
@@ -101,15 +103,10 @@ public void analyze(Analyzer analyzer) throws AnalysisException {
     }
 
     @Override
-    public SelectStmt toSelectStmt(Analyzer analyzer) throws AnalysisException {
+    public QueryStatement toSelectStmt() throws AnalysisException {
         if (where == null) {
             return null;
         }
-        if (selectStmt != null) {
-            return selectStmt;
-        }
-
-        analyze(analyzer);
```

Comment:
analyze function check the access permissions，could just delete it ?

Reply:
Under the new Analyzer system, the verification of permissions should be carried out in PrivilegeChecker, and there is no problem with deletion here. But unfortunately, we missed this point. I will submit another PR to resolve the statements currently supported by the new Parser and improve the code related to authorization authentication.